overlord/devicestate/firstboot.go

```diff
@@ -395,6 +400,10 @@ var loadDeviceSeed = func(st *state.State, sysLabel string) (deviceSeed seed.See
 		return nil, err
 	}
 
+	if runtimeNumCPU() > 1 {
+		deviceSeed.SetParallelism(2)
```

Comment:
This is not tested, is it?

Reply:
I would add some comment about:

// XXX set parallelism experimentally to 2 as I/O itself becomes a bottleneck ultimately

Reply:
In terms of testing we could have a test that mocks runtimeNumCPU to be 2. About checking that parallelism is set, one approach would be to look at timings and noticing the job spans?

Reply:
I added the comment, I think the test can come as a followup